SeriesCruncher/app/src/main/java/com/balazs_csernai/seriescruncher/app/component/ApplicationComponent.java

```diff
@@ -24,7 +26,8 @@
         modules = {
                 ApplicationModule.class,
                 ApiModule.class,
-                PreferenceModule.class
+                PreferenceModule.class,
+                AnimationModule.class
```

Comment:
As animations only used on screens and only activity has screens. We should set animation as common activity dependency. What's more since it is only used for details screen it would be grate if it was only in that component & module.